gsy-node/modules/gsy-collateral/src/mock.rs

```diff
@@ -69,18 +71,29 @@ parameter_types! {
 	pub const MaxLocks: u32 = 50;
 }
 
+impl pallet_timestamp::Config for Test {
+	type Moment = u64;
+	type OnTimestampSet = ();
+	type MinimumPeriod = ();
+	type WeightInfo = ();
+}
+
 impl pallet_balances::Config for Test {
```

Comment:
I have used this macro when implementing these pallets
#[derive_impl(pallet_balances::config_preludes::TestDefaultConfig)]